addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -1620,9 +1620,7 @@ const Wysiwyg = Widget.extend({
         }
     },
     _processAndApplyColor: function (eventName, color) {
-        if (!color) {
-            color = 'inherit';
-        } else if (!ColorpickerWidget.isCSSColor(color) && !weUtils.isColorGradient(color)) {
+        if (color && (!ColorpickerWidget.isCSSColor(color) && !weUtils.isColorGradient(color))) {
```

Comment:
@dmo-odoo Since we are removing the font tag altogether when we remove any color, I believe we don't need the 'inherit' property because we are not going to set it.